designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/UsageStatisticsReportsSettingsDeterminer.scala

```diff
@@ -109,12 +109,12 @@ class UsageStatisticsReportsSettingsDeterminer(
     fetchNonArchivedScenariosInputData: () => Future[List[ScenarioStatisticsInputData]]
 )(implicit ec: ExecutionContext) {
 
-  def determineStatisticsUrl(): Future[Option[String]] = {
+  def determineStatisticsUrl(): Future[Option[URL]] = {
```

Comment:
WDYT about changing the return type to `Either[Error, Option[URL]`?

Reply:
It's changed in this PR: https://github.com/TouK/nussknacker/pull/5867